appveyor.yml

```diff
@@ -33,9 +35,7 @@ after_build:
   - ps: nuget pack WinSW.nuspec -Version $env:APPVEYOR_BUILD_VERSION
 
 test_script:
-  - dotnet.exe test -f net40 --no-build src\Test\winswTests\winswTests.csproj
-  - dotnet.exe test -f net461 --no-build src\Test\winswTests\winswTests.csproj
-  - dotnet.exe test -f netcoreapp3.1 --no-build src\Test\winswTests\winswTests.csproj
+  - dotnet.exe test --no-build src\Test\winswTests\winswTests.csproj
```

Comment:
Will it trigger all configurations after that?

Reply:
Yes, as expected.